src/acceleration/Acceleration.hpp

```diff
@@ -64,9 +64,6 @@ class Acceleration {
   /// Checks if all dataIDs are contained in cplData
   void checkDataIDs(const DataMap &cplData) const;
 
-  /// Concatenates all coupling data involved into a single vector
-  void concatenateCouplingData(const DataMap &cplData, const std::vector<DataID> &dataIDs, Eigen::VectorXd &targetValues, Eigen::VectorXd &targetOldValues) const;
```

Comment:
I think we should keep this as `virtual void concatenateCouplingData(...) const = 0;` to make the relationship of `AitkenAcceleration::concatenateCouplingData` and `BaseQNAcceleration::concatenateCouplingData` clear. Compare to how some functions in the `BaseCouplingScheme` are implemented (e.g. [`hasAnySendData`](https://github.com/precice/precice/blob/15cdcbdf72fea5c06c996cc1c2e0f0b4e137916d/src/cplscheme/BaseCouplingScheme.hpp#L209))